app/src/features/surveys/view/components/SurveyStudyArea.tsx

```diff
@@ -72,14 +70,15 @@ const SurveyStudyArea: React.FC<ISurveyStudyAreaProps> = (props) => {
   const classes = useStyles();
   const biohubApi = useBiohubApi();
 
-  const {
-    projectForViewData,
-    surveyForViewData: {
-      surveyData: { survey_details },
-      surveySupplementaryData: { occurrence_submission }
-    },
-    refresh
-  } = props;
+  const surveyContext = useContext(SurveyContext);
+  const surveyForViewData = surveyContext.surveyDataLoader.data;
+
+  const { projectForViewData } = props;
+
+  const survey_details = useMemo(() => surveyForViewData?.surveyData?.survey_details, [surveyForViewData]);
+  const occurrence_submission = useMemo(() => surveyForViewData?.surveySupplementaryData?.occurrence_submission, [
+    surveyForViewData
+  ]);
```

Comment:
Is `useMemo` needed here? Since the value being memoized is also the same value in the dependencies, seems like it wouldn't add any benefit?

Reply:
As discussed, `useMemo` is used since the reference to `survey_details` would change every render. After testing, `useMemo` doesn't seem to affect the behaviour, so will remove.